ui/app/pages/settings/contact-list-tab/contact-list-tab.component.js

```diff
@@ -123,12 +99,7 @@ export default class ContactListTab extends Component {
     const { hideAddressBook, showingMyAccounts } = this.props;
 
     if (!hideAddressBook && !showingMyAccounts) {
```

Comment:
The `showingMyAccounts` prop is now always `false`, so it can be removed.

Similarly, the `MyAccounts` component that is used only when `showingMyAccounts` is `true` is also dead code and can be removed.